src/components/Input/inputBase/index.js

```diff
@@ -122,9 +122,11 @@ export default class InputBase extends Component {
             isCentered,
             iconPosition,
             variant,
+            valueAlignment,
         } = this.props;
         const isReadOnly = !!(!disabled && readOnly);
         const isPassword = type === 'password';
+        const alignment = isCentered ? 'center' : valueAlignment;
```

Comment:
here you are giving priority, to `isCentered` but since we will deprecate it, we should do the inverse, then let's give priority to the `valueAlignment` over the `isCentered`